.github/workflows/publish.yml

```diff
@@ -28,4 +28,4 @@ jobs:
           ORG_GRADLE_PROJECT_publish_packages_token: ${{ secrets.ORG_GRADLE_PROJECT_PUBLISH_PACKAGES_TOKEN }}
           ORG_GRADLE_PROJECT_signing_key : ${{secrets.ORG_GRADLE_PROJECT_SIGNING_KEY}}
           ORG_GRADLE_PROJECT_signing_password: ${{secrets.ORG_GRADLE_PROJECT_SIGNING_PASSWORD}}
-        run: ./gradlew publish
\ No newline at end of file
+        run: ./gradlew signMavenJavaPublication publishToSonatype closeAndReleaseSonatypeStagingRepository
```

Comment:
I thought this line was already present. But it is necessary for publishing to maven central.